.github/workflows/dockerimage-latest.yml

```diff
@@ -38,10 +37,8 @@ jobs:
         HUB_KEY: ${{ secrets.DOCKER_TOKEN }}
       run: |
         docker login -u $HUB_NAME -p $HUB_KEY
-        
-        docker build . --pull --no-cache --file Dockerfile.293LTS --tag crowdrender/blender-crowdrender:bl_2.93
-        
-        docker push crowdrender/blender-crowdrender:bl_2.93
+        docker build . --pull --no-cache --file Dockerfile.cuda10 --tag crowdrender/blender-crowdrender:bl_2.83_cu_10
```

Comment:
I know this is a relic of mine, but this will fail because `Dockerfile.cuda10` does not exist.

Reply:
I made a lot of changes here to support building multiple images. I've removed the reference to cuda10.